src/vcSettings.cpp

```diff
@@ -822,3 +824,16 @@ udResult vcSettings_UpdateLanguageOptions(vcSettings *pSetting)
 epilogue:
   return result;
 }
+
+void vcSettings_UpdateServerFlag(vcSettings *pSettings)
+{
+	pSettings->maptiles.serverFlag = vcMTSF_Unknown;
+	for (size_t i = 0; i < vcMTSF_Count; i++)
+	{
+		if (udStrstr(pSettings->maptiles.tileServerAddress, udStrlen(pSettings->maptiles.tileServerAddress), g_vcMapTileServerName[i]) != 0)
```

Comment:
You can't do this check. There are no rules for slippy servers; _we_ have `slippy` in our domain but no one else does.

You need to provide a combo box to select between Slippy (default) and other providers (like Google, Bing, etc). Those platforms need to know the users API KEY but don't need the domain.

[See this list](https://wiki.openstreetmap.org/wiki/Tile_servers) for domain names for other servers.

Reply:
Ok, I'll provide a combo box listing the available providers to choose from.The API KEY needs to be explored.

Reply:
This feature will create a new branch instead of adding it.

Reply:
This feature will recreate a branch and not be added to it.